.github/workflows/pre-commit.yml

```diff
@@ -53,7 +53,7 @@ jobs:
         working-directory: src/llama_stack_ui
 
       - name: Install pre-commit
-        run: python -m pip install pre-commit
+        run: python -m pip install pre-commit>=4.4.0
```

Comment:
you need to "quote" this correctly because that `>=` is getting interpreted by the shell